viewfield.module

```diff
@@ -114,9 +114,70 @@ function viewfield_field_formatter_info() {
       'label' => t('Default'),
       'field types' => array('viewfield'),
     ),
+    'viewfield_field_values' => array(
+      'label' => t('Field values'),
+      'field types' => array('viewfield'),
+      'settings' => array(
+        'fields_to_display' => array(
+          'vname' => 'vname',
+          'vdisplay' => 'vdisplay',
+          'vargs' => 'vargs',
+        ),
+      ),
+    ),
+
   );
 }
 
+/**
+ * Implements hook_field_formatter_settings_form().
+ */
+function viewfield_field_formatter_settings_form($field, $instance, $view_mode, $form, &$form_state) {
+  $display = $instance['display'][$view_mode];
+  $settings = $display['settings'];
+
+  $element = array();
+  $element['fields_to_display'] = array(
+    '#type' => 'checkboxes',
+    '#title' => t('Information to list'),
+    '#default_value' => $settings['fields_to_display'],
```

Comment:
This line appears to be the cause of the issue @laryn identified in [this comment](https://github.com/backdrop-contrib/viewfield/issues/1#issuecomment-1688389758). You may have to have warnings switched on to display. It may also be a PHP 8.1+ thing.

Anyway, this fixed it for me:

```suggestion
    '#default_value' => isset($settings['fields_to_display']) ? $settings['fields_to_display'] : array(),
```

Reply:
This looks reasonable.